vitest.config.e2e.ts

```diff
@@ -1,7 +1,7 @@
 import { resolve } from 'node:path'
 import { defineConfig } from 'vitest/config'
 
-const timeout = process.env.CI ? 50000 : 30000
+const timeout = process.env.PWDEBUG ? Infinity : process.env.CI ? 50000 : 30000
```

Comment:
I also added this to allow using playwright inspector without timeout.
For example, running this command will break at the first `page.goto(..)` and you can step debug and record with playwright browser window.

```sh
PWDEBUG=1 pnpm test-build playground/environment-react-ssr
```